roles/dcos_agent_windows/tasks/dcos_install.yml

```diff
@@ -5,19 +5,11 @@
 
 - name: Windows Installation | Download dcos_install.ps1
   win_get_url:
-    url: "{{ dcos['config']['bootstrap_url'] }}/{{ dcos['version'] }}/genconf_win/serve/prerequisites/dcos_install.ps1"
+    url: "{{ dcos['config']['bootstrap_url'] }}/{{ dcos['version'] }}/genconf/serve/windows/prerequisites/dcos_install.ps1"
     dest: "{{ base_windows_dir }}\\dcos_install.ps1"
 
 - name: Windows Installation | DC/OS install
   when: (ansible_local is not defined) or (ansible_local.dcos_installation is not defined) or (ansible_local.dcos_installation['dcos-image-commit'] is not defined)
   block:
     - name: Windows Installation | Run DC/OS agent installation
       win_shell: "{{ base_windows_dir }}\\dcos_install.ps1 '{{ dcos['config']['bootstrap_url'] }}' '{{ dcos['version'] }}' '{{ groups['masters'] | join(',') }}'"
-      register: install_shell_result
-      notify: reboot the host
-
-  always:
-    - name: Windows Installation | Output log into dcos-install.log
-      win_copy:
-        content: "{{ install_shell_result.stdout }}"
-        dest: "{{ base_windows_dir }}\\var\\log\\ansible_playbook_stdout.log"
```

Comment:
what is the reason not writing this log anymore?

Reply:
As far as I know `winpanda` is writing a similar file. But we should definitely also have a log-file that contains the ansible log output.

Reply:
let me roll this log write back. I thought it looks as redundant, since dcos_install as well as winpanda.py writes their own logs